From f654a25f9df6583532798f30181d149d51b23808 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Thu, 8 Aug 2019 08:34:47 +0300
Subject: [PATCH] Fix build on musl libc

Use bits/local_lim.h only with glibc as it's not available
on musl libc.

Change-Id: Ibffa4ab5649b544664f99b16e94d6865148eeeb5
---
 src/foundation/linux/Qt3DSLinuxThread.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/foundation/linux/Qt3DSLinuxThread.cpp b/src/foundation/linux/Qt3DSLinuxThread.cpp
index fa6e069..99c4c3a 100644
--- a/src/foundation/linux/Qt3DSLinuxThread.cpp
+++ b/src/foundation/linux/Qt3DSLinuxThread.cpp
@@ -35,7 +35,7 @@
 #include "foundation/Qt3DSAssert.h"
 #include "foundation/Qt3DSIntrinsics.h"
 #include "foundation/Qt3DSBroadcastingAllocator.h"
-#if !defined(QT3DS_APPLE) && !defined(ANDROID) && !defined(__CYGWIN__) && !defined(__QNX__) && !defined(__INTEGRITY)
+#if !defined(QT3DS_APPLE) && !defined(ANDROID) && !defined(__CYGWIN__) && !defined(__QNX__) && !defined(__INTEGRITY) && defined(__GLIBC__)
 #include <bits/local_lim.h> // PTHREAD_STACK_MIN
 #endif
 #include <stdio.h>
